### DISCUSSION TITLE
Simulation output for amount in large intestines  for a locally acting molecule that is not absorbed systemically

### DISCUSSION THREAD

**Discussion by mike2g** [Open-Systems-Pharmacology/Forum#860] [Questions & Problems] [answered]
I am working with PK Sim 9 and have added the building block of my simulations. The molecule I have is taken orally as a tablet but it is not absorbed systemically and will stay in the intestines and  eventually excreted in stool without change. It is locally acting in the large intestines. 
My PK Sim simulation only allows me to output (as chart or numerical values) the concentrations or amounts in the tissue of large intestines (ascending colon, traverse colon, etc.). It does not allow the amount or concentration that is unabsorbed in the lumen of the large intestines sections (for example amount at a given time om traverse colon). How can I get this output in my simulation?

Do I need to build an observer module in Mobi? I have not found information or guide to take me through an example. Any help or guidance is appreciated.

**Comment by StephanSchaller** [comment, marked as answer]
Hi @mike2g ,

you can directly output lumen concentrations in PK-Sim:
![image](https://user-images.githubusercontent.com/25477643/103520635-bd5d1180-4e77-11eb-8660-440cfe686d6c.png)

Hope this helps,
Stephan
